src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj

```diff
@@ -5,13 +5,23 @@
     <TargetFrameworks Condition="'$(UseIntrinsics)' == 'true'">netstandard2.0;netcoreapp3.0</TargetFrameworks>
     <IncludeInPackage>Microsoft.ML.CpuMath</IncludeInPackage>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <DefineConstants>$(DefineConstants);CORECLR;PRIVATE_CONTRACTS</DefineConstants>
```

Comment:
>CORECLR [](start = 40, length = 7)

you don't need this one.

Reply:
Why? I need to exclude some lines of Contract.cs in ML.Core.

Reply:
Sorry, should be more clear.(Although, I selected it).
I meant `CORECLR` constant. You don't need it.

---
In reply to: [236471797](https://github.com/dotnet/machinelearning/pull/1724#discussion_r236471797) [](ancestors = 236471797)

Reply:
Got it!

---
In reply to: [236473852](https://github.com/dotnet/machinelearning/pull/1724#discussion_r236473852) [](ancestors = 236473852,236471797)